### PR TITLE
Add page about new K3s `secrets-encrypt` tool.

### PR DESCRIPTION
**Pull request by dereknola**
Added detailed guides on how to use the new `k3s secrets-encrypt` CLI command for k3s. Moved existing info on secrets encryption into a dedicated page under the Security tab.

Also fixed typo on PostgreSQL.

### CODE REVIEW

**Review by jtravee — CHANGES_REQUESTED** [review]

**Review by jtravee — APPROVED** [review]

File: content/k3s/latest/en/security/secrets_encryption/_index.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+_Available as of v1.21.8+k3s1_
+
+K3s contains a utility tool `secrets-encrypt`, which enables automatic control over the following:
+
+- Disabling/Enabling secrets encryption
+- Adding new encryption keys
+- Rotating and deleting encryption keys
+- Reencrypting secrets
+
+>**Warning** Failure to follow proper procedure for rotating encryption keys can leave your cluster permanently corrupted. Proceed with caution.
```
Comment (jtravee) [inline content/k3s/latest/en/security/secrets_encryption/_index.md:56]:
```suggestion
>**Warning:** Failure to follow proper procedure for rotating encryption keys can leave your cluster permanently corrupted. Proceed with caution.
```

File: content/k3s/latest/en/security/secrets_encryption/_index.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+### Single-Server Encryption Key Rotation
+To rotate secrets encryption keys on a single-node cluster:
+
+- Start the K3s server with the flag `--secrets-encryption`
+
+>**Note** Starting K3s without encryption and enabling it at a later time is currently *not* supported.
```
Comment (jtravee) [inline content/k3s/latest/en/security/secrets_encryption/_index.md:63]:
```suggestion
>**Note:** Starting K3s without encryption and enabling it at a later time is currently **not** supported.
```

File: content/k3s/latest/en/security/secrets_encryption/_index.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+
+3. Reencrypt with flags
+
+    ```
+    k3s secrets-encrypt reencrypt --force --skip
+    ```
+
+### High-Availability Secrets Encryption Disable/Enable
+After launching a HA cluster with `--secrets-encryption` flags, secrets encryption can be disabled.
+>**Note** While not required, it is recommended that you pick one server node from which to run the `secrets-encrypt` commands.
```
Comment (jtravee) [inline content/k3s/latest/en/security/secrets_encryption/_index.md:158]:
```suggestion
>**Note:** While not required, it is recommended that you pick one server node from which to run the `secrets-encrypt` commands.
```

File: content/k3s/latest/en/security/secrets_encryption/_index.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+    ```
+    k3s secrets-encrypt reencrypt
+    ```
+
+### High-Availability Encryption Key Rotation
+The steps are the same for both embedded DB and external DB clusters.
+
+To rotate secrets encryption keys on HA setups:
+
+>**Note** While not required, it is recommended that you pick one server node from which to run the `secrets-encrypt` commands.
```
Comment (jtravee) [inline content/k3s/latest/en/security/secrets_encryption/_index.md:90]:
```suggestion
>**Note:** While not required, it is recommended that you pick one server node from which to run the `secrets-encrypt` commands.
```

File: content/k3s/latest/en/security/secrets_encryption/_index.md (hunk 1 above)
Comment (jtravee) [reply content/k3s/latest/en/security/secrets_encryption/_index.md:56]: Just a few more!

**Review by jtravee — CHANGES_REQUESTED** [review]

File: content/k3s/latest/en/security/secrets_encryption/_index.md (hunk 2 above)
Comment (dereknola) [reply content/k3s/latest/en/security/secrets_encryption/_index.md]: This breaks the formatting of the Note section if I try and bold **not** as well as **Note**
Only added the colon.

Comment (jtravee) [reply content/k3s/latest/en/security/secrets_encryption/_index.md]: Got it!

**Review by jtravee — APPROVED** [review]

**Pull request merged**